DXRMapFixups/DeusEx/Classes/DXRFixupM04.uc

```diff
@@ -284,6 +285,11 @@ function PreFirstEntryMapFixes()
                 GlowUp(key);
         }
 
+        if (#defined(injections) && VanillaMaps) {
+            compublic = Spawn(class'#var(prefix)ComputerPublic',,, vectm(741.36, 1609.34, 289.27), rotm(0, -16384, 0, 0));
+            compublic.BulletinTag = '03_BulletinMenu'; // same content that's in M03
```

Comment:
does this mean there could be updates to bingo goals and mission masks?

Reply:
Thank you, yes. I knew there was some bingo mask I wanted to change.